src/core/js/views/drawerView.js

```diff
@@ -8,8 +8,10 @@ define(function(require) {
         className: 'drawer display-none',
         disableAnimation: false,
         escapeKeyAttached: false,
+        forceCheckVisibility: false,
```

Comment:
also `forceCheckAvailability` would work better as a name given that its purpose is to trigger a call to `checkIfDrawerIsAvailable`